### DISCUSSION TITLE
Sequence of pages in menu

### DISCUSSION THREAD

**Discussion by Munz63** [requarks/wiki#4875] [Help / Questions] [answered]
Hi Team,
I'm establishing a new wiki and add pages. I figured out that pages which are on the same level are sorted alphabetically. However, I want to have them in a certain order. How can I do this? Is there an option to have numbering?
Many thanks!

**Comment by ranrinc** [comment]
I would also like to know how to do this, thanks

**Comment by NGPixel** [comment, marked as answer]
Use a Static or Custom Navigation modes to build your own menu. As for the browse / site tree mode, there's no way to sort them at the moment.
